core/eval.go

```diff
@@ -87,6 +89,29 @@ func evalSET(args []string) []byte {
 				return Encode(errors.New("ERR value is not an integer or out of range"), false)
 			}
 			exDurationMs = exDurationSec * 1000
+
+		case "PXAT", "pxat":
+			i++
+			if i == len(args) {
+				return Encode(errors.New("ERR syntax error"), false)
+			}
+
+			exDurationUnixMs, err := strconv.ParseInt(args[i], 10, 64)
+			if err != nil {
+				return Encode(errors.New("ERR value is not an integer or out of range"), false)
```

Comment:
Not a requirement for this PR, but it may be a good time to start consolidating the error messages for easier maintainability and ensuring that the error messages are exactly as that of REDIS.

Reply:
@soumya-codes please let me know I can create an issue and work on this if required.

Reply:
You can create the task for tracking purposes. Let's wait for Arpit and Jyotinder to see if this task needs to prioritized now over adding newer features. I am new to the project and I not aware of the current priorities.

Reply:
@soumya-codes Good point. We should start consolidating error messages. High time :)